### DISCUSSION TITLE
SNPs enrichment

### DISCUSSION THREAD

**Discussion by lukais-iohan** [GreenleafLab/ArchR#1671] [Questions / Documentation] [answered]
Hi,

I have a list of SNPs from a GWAS performed on a group of Alzheimer's patients. Does it possible to see if these SNPs are enriched in chromatin accesible regions with ArchR?

**Comment by rcorces** [comment, marked as answer]
We dont provide anything specifically to do this. We do have a new function on the `dev` branch to do hypergeometric enrichment but I think you might be better suited by something else.

https://github.com/GreenleafLab/ArchR/blob/3c650396ce45d497a9fe5fcb4015acad14882bc6/R/AnnotationPeaks.R#L1053-L1060

**Reply by lukais-iohan** [reply]
Thanks for your quick answer. I will try your suggestion.

Bests
